crates/rooch-framework/sources/address_mapping.move

```diff
@@ -70,12 +76,14 @@ module rooch_framework::address_mapping{
     /// Bind a rooch address to a multi-chain address
     public(friend) fun bind_no_check(ctx: &mut Context, rooch_address: address, maddress: MultiChainAddress) {
         if(multichain_address::is_rooch_address(&maddress)){
-            //Do nothing if the multi-chain address is a rooch address
-            return
+            assert!(
+                multichain_address::into_rooch_address(maddress) == rooch_address, 
+                error::invalid_argument(ErrorMultiChainAddressInvalid)
+            );
```

Comment:
Add return here? Otherwise, the AddressMapping will add a `rooch address` -> `rooch MultiChainAddress` record.

Reply:
I merged this PR first, @pause125 you can fix this in new PR